explore-analyze/machine-learning/nlp/ml-nlp-model-ref.md

```diff
@@ -15,7 +15,7 @@ products:
 The minimum dedicated ML node size for deploying and using the {{nlp}} models is 16 GB in {{ech}} if [deployment autoscaling](../../../deploy-manage/autoscaling.md) is turned off. Turning on autoscaling is recommended because it allows your deployment to dynamically adjust resources based on demand. Better performance can be achieved by using more allocations or more threads per allocation, which requires bigger ML nodes. Autoscaling provides bigger nodes when required. If autoscaling is turned off, you must provide suitably sized nodes yourself.
 ::::
 
-The {{stack-ml-features}} support transformer models that conform to the standard BERT model interface and use the WordPiece tokenization algorithm.
+The {{stack-ml-features}} support transformer models that conform to the standard BERT model interface and use the WordPiece tokenization algorithm. {{stack-ml-features}} will always use the non-fast ("slow") tokenizer variant for all supported models. This ensures deterministic and stable tokenization results across different platforms and avoids potential differences in handling between fast and slow implementations.
```

Comment:
The reference to the "standard BERT model inferface" and "WordPiece are not clear to the average reader. I think it's better just to go into the list of supported architectures

```suggestion
The {{stack-ml-features}} support transformer models with the following architectures:
```

Then addd the new line after the list of supported architectures at line 34.
```
{{stack-ml-features}} will always use the non-fast ("slow") tokenizer variant for all supported models. This ensures deterministic and stable tokenization results across different platforms and avoids potential differences in handling between fast and slow implementations.
```